### ISSUE TITLE
Courses in Common doesn't work at all if you can't see the members of any course

### ISSUE THREAD

**Issue by ajs256** [aopell/SchoologyPlus#275]
### Describe the bug
Courses in Common doesn't work if you can't see the members of a course.

### Steps to reproduce
1. Be in a course where the ability to see members is turned off. (not sure what the teacher-side setting is called)
2. Go to anyone's profile
### Expected behavior
It works

### Additional context
Log:
```
+ Error building courses in common:  
headers: {}
json: () => readBodyTask
ok: false
redirected: false
status: 403
statusText: ""
type: "basic"
url: "https://api.schoology.com/v1//sections/REMOVED/enrollments"
[[Prototype]]: Object
```

### Platform

Chromium ungoogled-chromium 97.0.4692.71
fuhsd.schoology.com
S+ 7.3.5

**Comment by aopell** [comment]
Unfortunately there's probably no way for us to get around this -- if you're unable to see the members of the course, we can't check who's in it

**Comment by ajs256** [comment]
I would like to still be able to see the courses in common, just ignoring this one

**Comment by aopell** [comment]
Fixed in 6990318
